hazelcast/src/main/java/com/hazelcast/internal/monitor/impl/LocalMapStatsImpl.java

```diff
@@ -115,36 +144,36 @@
     private volatile long maxPutLatency;
     private volatile long maxSetLatency;
     private volatile long maxRemoveLatency;
-    @Probe
+    @Probe(name = MAP_METRIC_CREATION_TIME)
     private volatile long creationTime;
-    @Probe
+    @Probe(name = MAP_METRIC_OWNED_ENTRY_COUNT)
     private volatile long ownedEntryCount;
-    @Probe
+    @Probe(name = MAP_METRIC_BACKUP_ENTRY_COUNT)
     private volatile long backupEntryCount;
-    @Probe
+    @Probe(name = MAP_METRIC_OWNED_ENTRY_MEMORY_COST)
```

Comment:
How about `BYTES` unit here and in other memory related probes in this class?

Reply:
Done